backup/backup.go

```diff
@@ -26,7 +26,7 @@ func NewBucketManifestWriter(ts *tenant.Service, mc *meta.Client) BucketManifest
 // It is intended to be used to write to an HTTP response after appropriate measures have been taken
 // to ensure that the request is authorized.
 func (b BucketManifestWriter) WriteManifest(ctx context.Context, w io.Writer) error {
-	bkts, _, err := b.ts.FindBuckets(ctx, influxdb.BucketFilter{})
```

Comment:
Is there a reason FindBuckets is limited by default? I think ideally FindBuckets would return everything unless limit/filter are applied by the caller, rather than hard-limiting everything and requiring this ugly workaround.

Reply:
It has the following comment in it, from several years ago. I don't know if it was in response to performance/other complaints or something done preemptively.

`// if we dont have any options it would be irresponsible to just give back all buckets in the system
`